lib/interfaces/gql-module-options.interface.ts

```diff
@@ -1,19 +1,26 @@
 import { Type } from '@nestjs/common';
 import { ModuleMetadata } from '@nestjs/common/interfaces';
 import {
-  Config,
-  IResolverValidationOptions,
-  ServerRegistration,
+  Config as ConfigExpress,
+  IResolverValidationOptions as IResolverValidationOptionsExpress,
+  ServerRegistration as ServerRegistrationExpress,
 } from 'apollo-server-express';
+
+import {
+  Config as ConfigFastify,
+  IResolverValidationOptions as IResolverValidationOptionsFastify,
+  ServerRegistration as ServerRegistrationFastify,
+} from 'apollo-server-express';
```

Comment:
I suppose that you wanted to import from `apollo-server-fastify` here?

Reply:
@kamilmysliwiec will fix this :) thanks for noticing

Reply:
fixed